pysatNASA/instruments/icon_euv.py

```diff
@@ -39,6 +39,7 @@
 Authors
 ---------
 Jeff Klenzing, Mar 17, 2018, Goddard Space Flight Center
+
 Russell Stoneback, Mar 23, 2018, University of Texas at Dallas
```

Comment:
I'm open to either, but the formatting in RTD is weird otherwise.
With a blank line:
<img width="575" alt="Screen Shot 2022-04-21 at 5 06 44 PM" src="https://user-images.githubusercontent.com/19592220/164552534-d5194d53-f068-4995-b520-11483b129df3.png">

Without:
<img width="669" alt="Screen Shot 2022-04-21 at 5 07 23 PM" src="https://user-images.githubusercontent.com/19592220/164552570-357216ba-9fa4-4bad-b83d-ca27d7e2683a.png">

Reply:
misread that.  I can remove the authors sections throughout